composer.json

```diff
@@ -28,7 +28,8 @@
         "php": ">=5.4",
         "symfony/console": "^2.8|^3.0|^4.0",
         "symfony/config": "^2.8|^3.0|^4.0",
-        "symfony/yaml": "^2.8|^3.0|^4.0"
+        "symfony/yaml": "^2.8|^3.0|^4.0",
+        "josegonzalez/dotenv": "2.*"
```

Comment:
Does anyone think this should be require-dev as optional one for those that do not use/want it?

Reply:
The migrations will probably be run in every environment, not just while developing, right? In which case, I think this should be a requirement, not a dev requirement. Just my 2 cents but I maybe wrong in my understanding.

Reply:
A dev requirement (along with suggest text) means that the functionality is there but dormant.
Once you require it in your root level (project application), it then becomes a real functionality.
So no, this should be up to the dev IMO. Along with a documentation on how to enable this of course.

Reply:
OK :)

Reply:
Changed.